examples/source/1.components/amp-script/index.html

```diff
@@ -219,6 +218,45 @@
       </script>
     </div>
 
+    <!--
+      ## Data source for `amp-list`
+      An `<amp-script>` function can serve as the data source for an `<amp-list>`.
+        - Use `exportFunction()` to make the function visible to the `<amp-list>`.
+        - Specify the script and the function in the `src` attribute of the `<amp-list>`. Use the form `src="amp-script:{scriptID}:functionName"`, where `{scriptID}` is the `id` of the `<amp-script>` and `{functionName}` is the name of the exported function.
+        - You can use the `nodom` attribute in the `<amp-script>` to indicate that the `<amp-script>` won't need a DOM. This improves performance, because `amp-script` doesn't need to load or execute its virtual DOM implementation.
+    -->
+    <div class="sample">  
+      <amp-script id="dataFunctions" script="protocol-adapter-script" nodom></amp-script>
+      <script id="protocol-adapter-script" type="text/plain" target="amp-script">
```

Comment:
nit: I'm not sure if we want to keep the name "protocol adapter". It constrains the capability to `fetch`/xhr, whereas the JS can basically do anything (localStorage, websockets, math computations, etc.). Maybe `<script id="dataFunctions-script">`

Reply:
made id even more explicit!